examples/nlp/Tweet-classification-using-TFDF.py

```diff
@@ -0,0 +1,272 @@
+"""
+Title: Text Classification using TFDF and Pre-trained embeddings
+Author: Gitesh Chawda
+Date created: 30/04/2022
+Last modified: 30/04/2022
+Description: Using Tensorflow Decision Forest for text classification
+"""
+
+"""
+## Introduction
+
+[TensorFlow Decision Forests](https://www.tensorflow.org/decision_forests) (TF-DF)
+provide powerful models, especially with structured data,
+It is a collection of state-of-the-art algorithms of Decision Forest models that are
+compatible with Keras APIs. The module includes Random Forests, Gradient Boosted Trees,
+and CART, and can be used for regression, classification, and ranking tasks.
+
+Alternatively for getting started you go through official tutorial 
+[beginner Colab](https://www.tensorflow.org/decision_forests/tutorials/beginner_colab) 
+also text classification using pre-trained embeddings
+[Notebook](https://www.tensorflow.org/decision_forests/tutorials/intermediate_colab).
+
+In this example we will use Gradient Boosted Trees with pre-trained embeddings to
+classify disaster tweets.
+"""
+
+"""
+Install Tensorflow Decision Forest using following command : 
+`!pip install tensorflow_decision_forests`
+"""
+
+"""
+## Imports
+"""
+
+import pandas as pd
+import numpy as np
+import tensorflow as tf
+from tensorflow import keras
+import tensorflow_hub as hub
+from tensorflow.keras import layers
+import tensorflow_decision_forests as tfdf
+import matplotlib.pyplot as plt
+
+"""
+## Get the data
+
+Dataset is avaliable at [kaggle](https://www.kaggle.com/c/nlp-getting-started)
+
+dataset description :
+
+1. Files
+    - train.csv - the training set
+
+2. Columns
+    - id - a unique identifier for each tweet
+    - text - the text of the tweet
+    - location - the location the tweet was sent from (may be blank)
+    - keyword - a particular keyword from the tweet (may be blank)
+- target - in train.csv only, this denotes whether a tweet is about a real disaster (1)
+or not (0)
+"""
+
+# Turn .csv files into pandas DataFrame's
+df = pd.read_csv(
+    "https://raw.githubusercontent.com/IMvision12/Tweets-Classification-NLP/main/train.csv"
+)
+print(df.head())
+
+# Printing shape of the training dataset
+print(f"Training dataset shape: {df.shape}")
+
+# Shuffling training data
+df_shuffled = df.sample(frac=1, random_state=42)
+# Dropping id, keyword and location columns as these columns consists of mostly nan values
+# we will be using only text and target columns
+df_shuffled.drop(["id", "keyword", "location"], axis=1, inplace=True)
+df_shuffled.reset_index(inplace=True, drop=True)
+print(df_shuffled.head())
+
+print(df_shuffled.info())
+
+# Printing total number of Disaster and non-Disaster tweets
+print(
+    f"Total Number of disaster and non-disaster tweets\n{df_shuffled.target.value_counts()}"
+)
+
+# Viewing 5 records from training data
+for ind, counter in enumerate(df_shuffled.index):
```

Comment:
Nit: Replace block with:

for index, example in df[:5].iterrows():
  print(f"Example #{index}")
  print(f"\tTarget : {example['target']}")
  print(f"\tText : {example['text']}")